packages/shared/styles/components/SfCollectedProduct.scss

```diff
@@ -0,0 +1,68 @@
+@import "../variables";
+
+$collected-product-hover-shadow: 0px 4px 35px rgba(168, 172, 176, 0.19) !default;
+$collected-product__remove-bg: $c-gray-secondary !default;
+.sf-collected-product {
+  max-width: 100%;
+  position: relative;
+  transition: 0.3s all;
```

Comment:
what is a purpose of this property?

Reply:
i use to keep the relative component container where it is used